docs/UserGuide.md

```diff
@@ -55,283 +65,419 @@ Choose a topic from the table of contents to get started on your Wishful Shrinki
+  e.g. `[t/TAG]…​` can be used as `t/healthy` or `t/healthy t/low calories`.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME i/INGREDIENTS`, `i/INGREDIENTS n/NAME` is also acceptable.
 </div>
+<br><br>
+
+## Main Commands
+
+The main commands include `help` and `exit`. They are the main commands you can use in Wishful Shrinking that are separate from the Recipes, Fridge and Consumption list.
+<br><br><br>
 
 ### Viewing help : `help`
 
 Shows a message explaining how to access the help page.
 
+<br><br>
 ![help message](images/helpMessage.png)
+<br>The image above is Wishful Shrinking's help window.<br><br><br>
 
 Format: `help`
+<br><br><br>
 
 ### Exiting the program : `exit`
 
 Exits the program.
 
 Format: `exit`
+<br><br><br>
+
+## Recipe-related Commands
 
-## Recipe
+The Recipe-related commands include `addR`, `recipes`, `selectR`, `close`, `deleteR`, `editR`, `searchR`, `recommend` and `clearR`. These are the commands in Wishful Shrinking that are relevant only to the Recipes.
+<br><br><br>
 
 ### Adding a recipe: `addR`
 
-Adds a recipe to Recipes list.
+Adds a recipe to Recipe list.
 
-Format: `addR n/TITLE i/INGREDIENT [ -QUANTITY][, MORE INGREDIENTS [ -QUANTITY]] c/CALORIES img/IMAGE inst
-/INSTRUCTION... [t/TAG]...`
+<br><br>
+<img src="images/AddRecipeImage.png" width="550" height="300">
+<br>  The image above is what Wishful Shrinking looks like after you have added a recipe.<br><br><br>
 
-* `INGREDIENT` can take in an optional `Quantity` e.g. i/Tomato -2 whole
-    * there is a mandatory space before `-` 
+Format: `addR n/NAME i/INGREDIENT [ -QUANTITY][, MORE INGREDIENTS [ -QUANTITY]] c/CALORIES img/IMAGE inst/INSTRUCTION... [t/TAG]...`
+
+* `INGREDIENT` can take in an optional `Quantity` e.g. i/Tomato -2 whole.
+    * Note: there is a mandatory space before `-`.
+* `CALORIES` **must be a positive integer** e.g. 100, 200, 300......
 * `IMAGE` can be local path e.g. images/healthy1.jpg or url e.g. https://vaya.in/recipes/wp-content/uploads/2018/06/Club-sandwich.jpg
+   * Note: If you want to use an image that has been saved in your computer, you have to specify the **absolute path** of the image, and add file:// in front of the file path. Then, replace IMAGE in img/IMAGE with the file path of your image, e.g. img/file:///D:/myimages/wishful/data/myimage.png.
+   * Note: If you input an invalid file path, Wishful Shrinking will replace the image with a default image.
+   * Note: If you want to use a url as the image, make sure that you are connected to the internet on your computer, and that the url is the **image address** (starting with https:// and ending with .jpg/jpeg/png). The image will be downloaded into Wishful Shrinking's data folder if there is internet connection.
+   * Note: If you input an invalid url, or input a url while there is no internet, Wishful Shrinking cannot add the image to the recipe. You have to connect to the internet, or check that your url is a valid image address.
```

Comment:
Should we include a table of built in image paths, e.g. images/healthy1.jpg together with their respective descriptions, just in case users want to use our sample images?

Reply:
for the last Note, if input invalid url/valid url without internet, Wishful Shrinking will use default image instead.